voter/migrations/0029_make_age_numeric.py

```diff
@@ -0,0 +1,48 @@
+# data->>'age' has been imported as text, should be integer
+
+from django.db import migrations, connection
+
+
+def update_age(apps, schema):
+    NCVoter = apps.get_model('voter.NCVoter')
+
+    # Doing these individually would be slow. Maybe we can finesse it.
+
+    # Here's some SQL to update a batch at once (those with age '20'):
+    #
+    # update voter_ncvoter set data = data || '{"age": 20}' where data->>'age' = '20';
+    #
+    # I have not been able to come up with SQL that will update all records in one
+    # go, but we can iterate pretty easily over the range of reasonable ages
+    # and do this in 150 batches rather than individually to millions of records.
+
+    with connection.cursor() as cursor:
+        for age in range(150):
+            sql = """update voter_ncvoter set data = data || '{"age": %s}' where data->>'age' = '%s';""" % (age, age)
+            cursor.execute(sql)
+            print("Age: %d, updated rows: %d" % (age, cursor.rowcount))
```

Comment:
If we need to update data in the system has already importer, they we also need to update the importer for new data that comes in, don't we?

Reply:
True. That change was made on the previous pull request and I accidentally left it there when I was separating them out.  Look at parse_row on the NCVoter model.

Reply:
👍